task4/task4.ipynb

```diff
@@ -112,41 +131,17 @@
     "            retention_rates_all_cohorts[j][i] = np.NaN\n",
     "    return retention_rates_all_cohorts\n",
     "\n",
-    "get_all_retention_rates_months_after_acquisition = get_all_retention_rates_time_normalized(data)\n",
+    "retention_rates_cohort0 = np.array(get_all_retention_rates_for_cohort(data,0))\n",
```

Comment:
Using changing retention rates for cohort 0 only instead of the average changing retention rates